src/haskell/parser/src/KoreAST.hs

```diff
@@ -0,0 +1,229 @@
+module KoreAST where
```

Comment:
An exports list would help to discern what is public and what is internal. If everything is public then maybe write a comment about it?

Reply:
we thought everything here would be public.  But you are right about a comment. Will do